### PR TITLE
HeroBanner Added in Both Mobile and Desktop

### PR DESCRIPTION
**Pull request by Monilprajapati**
Created Hero Banner for both Mobile and desktop devices.
Issue#14 solved About Hero Section

### CODE REVIEW

**Comment by victorchrollo14** [comment]
@Monilprajapati .Looks good on mobile, try and add a clamp for fontsize for that text in hero section so that increases till 600px, Then break it at sm: to look like desktop design.

File: client/src/pages/home/Home.jsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ import img4 from '../../assets/sweater.webp'
 import img5 from '../../assets/accesories.webp'
 import img6 from '../../assets/footwear.webp'
 
+import Footer from "../../components/Footer";
```
Comment (victorchrollo14) [inline client/src/pages/home/Home.jsx:12]: remove this line since, it's already imported and causing an error.

**Review by victorchrollo14 — CHANGES_REQUESTED** [review]
just fix these, that's all.

File: client/src/pages/home/Home.jsx (hunk 1 above)
Comment (Monilprajapati) [reply client/src/pages/home/Home.jsx:12]: Okkkay will fix it

**Review by victorchrollo14 — APPROVED** [review]
looks good.

**Pull request merged**